RetrospectiveExtension.Frontend/components/__mocks__/mocked_components/mockedFeedbackColumn.tsx

```diff
@@ -90,7 +89,7 @@ testColumnsObj[testColumnUuidOne] = {
 testColumnsObj[testColumnUuidTwo] = {
   columnProperties:
   {
-    id: TooltipOverflowMode,
```

Comment:
I don't know why this was using this random import, but it seemed like a bug.

Reply:
Appreciate the diligence! Always appreciated :D